ruby/secret_agents.rb

```diff
@@ -0,0 +1,115 @@
+#secret agent encryption
+
+=begin
+encryption (pseudocode)
+
+-define the method (argument)
+	-define index variable
+	-define empty return string variable
+	-create a loop that cycles through each index until remaining character
+		-if it's a letter (if it's NOT a space)
+			-advance one letter forward
+			-add to return string
+		-if "z"
+	    	-add "a" to return string
+		-end
+	-end loop
+	-return the full return string
+-end method
+=end
+
+#The encrypt method returns the next letter in the alphabet for each inputed value in string.
+def encrypt str
+  index = 0
+  return_str = ""
+  until index >= str.length
+    if str[index] != " " && str[index] != "z"
```

Comment:
@bontgoy so what you're saying is simplifying the control flow portion so as to not repeat my conditions as much. in terms of order of operations, what do you mean exactly?